backend/models.py

```diff
@@ -13,78 +12,83 @@ def __str__(self) -> str:
 
 
 class UploadRequest(models.Model):
-    user = models.ForeignKey(get_user_model(), on_delete=DO_NOTHING,
+    user = models.ForeignKey(get_user_model(), on_delete=models.DO_NOTHING,
                              null=True, blank=True)
 
 
 class FieldOfStudy(models.Model):
     DEGREE = (
-        ("7", "7 terms I"),
-        ("6", "6 terms I"),
-        ("3", "3 terms II"),
-        ("4", "4 terms II")
+        ("1", "Bachelor"),
+        ("2", "Master")
     )
+
+    MODE = (
+        ("stacjonarne", "stacjonarne"),
+        ("niestacjonarne", "niestacjonarne")
+    )
+
     faculty = models.ForeignKey(Faculty, on_delete=models.CASCADE)
     name = models.CharField(max_length=150)
-    degree = models.CharField(choices=DEGREE, max_length=10,
-                              null=True, blank=True)
+    degree = models.CharField(choices=DEGREE, max_length=10, null=True,
+                              blank=True, default="1")
+    type = models.CharField(max_length=100, null=True, blank=True,
+                            default="stacjonarne")
 
 
 class Candidate(models.Model):
-    upload_request = models.ForeignKey(UploadRequest, on_delete=CASCADE,
+    upload_request = models.ForeignKey(UploadRequest, on_delete=models.CASCADE,
                                        null=True, blank=True)
-    first_name = models.CharField(max_length=50)
-    last_name = models.CharField(max_length=50)
-    date_of_birth = models.DateField()
-    gender = models.CharField(max_length=50)
-    year_of_exam = models.IntegerField()
-    city = models.CharField(max_length=80)
-    contest = models.CharField(max_length=80, null=True)
+    pesel = models.CharField(max_length=50, null=True, blank=True)
+    first_name = models.CharField(max_length=50, null=True, blank=True)
+    last_name = models.CharField(max_length=50, null=True, blank=True)
+    date_of_birth = models.DateField(null=True, blank=True)
+    gender = models.CharField(max_length=50, null=True, blank=True)
+    city = models.CharField(max_length=80, null=True, blank=True)
+    contest = models.CharField(max_length=80, null=True, blank=True)
```

Comment:
To nie powinny być null, imo

Reply:
A tu akurat nie wiem. Te dane nie mają kompletnie żadnego znaczenia dla nas (oprócz olimpiady - ale nie każdy musi ją mieć i np. peselu żeby się dało odróżnić ludzi), a szkoda żeby plik nie przechodził z powodu braku informacji o imieniu.
Sugerowałem się też tym co klient miał gdzieś i zostawił puste pola w przykładowym pliku.